src/libraries/Common/src/Interop/Unix/System.Native/Interop.InterfaceNameToIndex.cs

```diff
@@ -10,5 +10,8 @@ internal static partial class Sys
     {
         [LibraryImport(Libraries.SystemNative, EntryPoint = "SystemNative_InterfaceNameToIndex", StringMarshalling = StringMarshalling.Utf8, SetLastError = true)]
         public static partial uint InterfaceNameToIndex(string name);
```

Comment:
Is the string-based one still used?

Reply:
No. The same goes for `if_nametoindex`, and both native methods' calling methods in `InterfaceInfoPal`. I've removed these too.